### DISCUSSION TITLE
Speaker Count Only?

### DISCUSSION THREAD

**Discussion by MrEdwards007** [pyannote/pyannote-audio#1670] [General]
What can be done to obtain the count of speakers only.  

Currently the entire pipeline process is running at Task Speed: 2.105x (GPU enabled), so for a 23 minute audio file, it takes just a little over 10 minutes for the entire process, which includes segmentation, speaker counting, clustering and discrete diarization. I am only interested in the number of speakers.  

How is this achievable at the highest speed possible?

**Comment by SujithKumar01** [comment]
Can we get the speaker count in an audio ?
